WordPress/Classes/ViewRelated/Views/AlertView/AlertView.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+
+
+/**
+*  @class           AlertView
+*  @brief           The purpose of this class is to render a simple AlertView, with a custom style,
+*                   and the following sections: [ Title, Message, Button ]
+*/
+
+public class AlertView : NSObject
+{
+    // MARK: - Public Aliases
+    public typealias Completion = (() -> ())
+
+    
+    /**
+    *  @details     Designated Initializer
+    *  @param       title       The title of the AlertView.
+    *  @param       message     Message string to be displayed. Note: Bold is supported, **markdown flavor**.
+    *  @param       completion  A closure to be executed right after the button is pressed.
+    */
+    public init(title: String, message: String, button: String, completion: Completion?) {
+        super.init()
+        
+        // Load the nib
```

Comment:
Try to avoid using comments to narrate the code as it duplicates the number of lines to be read for no gain.  I'd suggest removing all comments in this method since they add no information to the reader.